.buildkite/pipeline.yml

```diff
@@ -1,13 +1,27 @@
 steps:
-  - label: 'check-hydra'
-    command: 'ci/check-hydra.sh'
+  - label: 'hydra-eval-errors'
+    command: 'nix-build ./nix -A iohkNix.hydraEvalErrors && ./result/bin/hydra-eval-errors.py'
     agents:
       system: x86_64-linux
-  - label: 'stack2nix'
-    command: 'ci/check-stack2nix.sh'
+
+  - label: 'stack rebuild'
+    env:
+      STACK_ROOT: "/build/ouroroboros-network.stack"
+    command:
+      - "rm -rf /build/ouroroboros-network"
+      - "cp -R . /build/ouroroboros-network"
+      - "cd /build/ouroroboros-network"
+      - "nix-build scripts/buildkite -o stack-rebuild"
+      - "./stack-rebuild"
     agents:
       system: x86_64-linux
-  - label: 'dependencies-in-master'
-    command: 'ci/check-dependencies-merged-in-master.sh'
```

Comment:
Why is this gone?

Reply:
There is no .stack.nix directory anymore. We may need to see if there's a better way to check this.